DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/function.json

```diff
@@ -6,7 +6,7 @@
       "type": "timerTrigger",
       "runOnStartup": false,
       "direction": "in",
-      "schedule": "0 */60 * * * *"
+      "schedule": "0 */5 * * * *"
```

Comment:
Lets make sure the frequency of polling is double the frequency of Analytic rule

Reply:
Yes, I will make it 10 minutes.